core/ecschema-metadata/src/Metadata/Class.ts

```diff
@@ -25,7 +25,7 @@ import { ECSpecVersion, SchemaReadHelper } from "../Deserialization/Helper";
 
 /**
  * A common abstract class for all of the ECClass types.
- * @beta
+ * @public @preview
  */
 export abstract class ECClass extends SchemaItem implements CustomAttributeContainerProps {
   public static override get schemaItemType(): SupportedSchemaItemType { return AbstractSchemaItemType.Class; } // need this so getItem("name", ECClass) in schema works
```

Comment:
Does the static schemaItemType need to be public?  Don't we just use it internally?  Can it be marked @internal?

Reply:
I ended up marking all protected things internal.  We were very inconsistent with how we marked protected things.

Next question should we mark SchemaItem constructors as internal? I think so because they should only ever be called through the editor right?

Reply:
Sounds reasonable. Those protected members should never be touched by anyone other than our editing API.